### PR TITLE
Package fsml.0.2.1

### PR DESCRIPTION
**Pull request by jserot**
### `fsml.0.2.1`
A library for describing and describing synchronous finite state machines

---
* Homepage: http://github.com/jserot/fsml
* Source repo: git+https://github.com/jserot/fsml.git
* Bug tracker: jocelyn.serot@uca.fr

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by mseri** [comment]
I think you need to include the opam file in the tarball: 
```
#=== ERROR while compiling fsml.0.2.1 =========================================#
# context     2.0.7 | macos/x86_64 | ocaml-system.4.10.0 | file:///Users/travis/build/ocaml/opam-repository
# path        ~/.opam/ocaml-system/.opam-switch/build/fsml.0.2.1
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune build -p fsml -j 1 @install @runtest
# exit-code   1
# env-file    ~/.opam/log/fsml-1639-42bb52.env
# output-file ~/.opam/log/fsml-1639-42bb52.out
### output ###
# Error: I don't know about package fsml (passed through -p)
```

**Comment by jserot** [comment]
Fixed

File: packages/fsml/fsml.0.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/jserot/fsml.git"
```
Comment (mseri) [inline packages/fsml/fsml.0.2.1/opam:31]: It is missing the url section pointing at the sources here
```suggestion
dev-repo: "git+https://github.com/jserot/fsml.git"
url {
  src: "https://github.com/jserot/fsml/archive/0.2.1.tar.gz"
  checksum: "sha512=d649da27e4f08caa043736523c62a357df951c79a5589cd1e2d5c264540d0da6088a2b24bb51401cb54f6be79444bcdac86bccb496463deb75b4646b35cd7ba2"
}
```
Maybe you need to update opam-publish, but sometimes it is just not picking it up -- no idea why.

Comment (mseri) [reply packages/fsml/fsml.0.2.1/opam:31]: Anyways, this should do, no action needed.

**Comment by mseri** [comment]
It is good now, finally! Thanks

**Comment by mseri** [comment]
Feel free to announce the release on the discuss.ocaml.org forum. We have a community category and a announcement tag for this purpose.

**Comment by jserot** [comment]
Thanks for your help !

**Pull request merged**